src/main/java/uk/gov/hmcts/reform/sscs/service/evidence/EvidenceUploadService.java

```diff
@@ -364,22 +371,30 @@ private void updateCaseDataWithNewAudioVideoUpload(List<SscsDocument> audioVideo
         String originalSenderPartyName = getOtherPartyName(sscsCaseData, withEmailPredicate(idamEmail));
 
         for (SscsDocument audioVideoDocument : audioVideoMedia) {
-
+            SscsDocumentDetails sscsDocumentDetails = audioVideoDocument.getValue();
             audioVideoEvidence.add(AudioVideoEvidence.builder()
-                    .value(AudioVideoEvidenceDetails.builder()
-                            .documentLink(audioVideoDocument.getValue().getDocumentLink())
-                            .dateAdded(ldt.toLocalDate())
-                            .fileName(audioVideoDocument.getValue().getDocumentFileName())
-                            .partyUploaded(uploader)
-                            .originalSenderOtherPartyId(originalSenderOtherPartyId)
-                            .originalSenderOtherPartyName(originalSenderPartyName)
-                            .statementOfEvidencePdf(draftSscsDocument.getValue().getDocumentLink())
-                            .build())
-                    .build());
+                .value(AudioVideoEvidenceDetails.builder()
+                    .originalSenderOtherPartyId(originalSenderOtherPartyId)
+                    .originalSenderOtherPartyName(originalSenderPartyName)
+                    .documentLink(sscsDocumentDetails.getDocumentLink())
+                    .dateAdded(ldt.toLocalDate())
+                    .fileName(sscsDocumentDetails.getDocumentFileName())
+                    .partyUploaded(uploader)
+                    .documentType(AudioVideoEvidenceUtil.getDocumentTypeValue(
+                        sscsDocumentDetails.getDocumentLink().getDocumentFilename()))
+                    .statementOfEvidencePdf(draftSscsDocument.getValue().getDocumentLink())
+                    .build())
+                .build());
+            System.out.println(sscsDocumentDetails.getDocumentLink().getDocumentFilename());
```

Comment:
Remove this System.out.println

Reply:
Oops, debugger isn't working so having to use those